### ISSUE TITLE
inaccessible histories

### ISSUE THREAD

**Issue by IonEwe** [swift-kim/SongPlayHistory#8]
i have over a thousand histories that aren't accessible
1) have score on score saber
2) old and "new" maps
3) history file seems to be there but doesn't show in game
4) lists as "uplayed"
5) time past play become inacessible days to months. not sure.
6) present in versions 1.18 to 1.25.x, so it has been happening for years.
i have ~ 5k plays... would like to resolve this before i remove my self from public databases. (fixing old plays is not the goal but would be nice)
not sure how to even diagnose what is going on with this.

**Comment by swift-kim** [comment]
I'm not sure what you mean by "aren't accessible" but if it's caused by the plugin crashing during game startup you may first check the plugin logs. The log files are located at the `Logs` directory in your game installation path. Also make sure the plugin you downloaded from https://github.com/Shadnix-was-taken/BeatSaber-SongPlayHistoryContinued/releases matches your game version.

**Comment by IonEwe** [comment]
the history shows  "-" instead of a score (SS shows a score). if the plug in crashed... no histories would not would display? in the case of this issue some show... some don't. I'm using the most recent release and it is still happening. maybe i should take a image to explain?

**Comment by swift-kim** [comment]
![image](https://user-images.githubusercontent.com/40190588/203835189-b5ea5184-f0e3-48b3-aae5-18b4bfb00360.png)

In which part do you have the problem? If it's 2, your Beat Saber data file (`%AppData%\..\LocalLow\Hyperbolic Magnetism\Beat Saber\PlayerData.dat`) might have been corrupted.

**Comment by IonEwe** [comment]
yes, part #2.  how do i verify PlayerDat.dat? i have backups if needed. Part 1 is also empty (on the ones i have checked). Given the thousands of examples i might find examples of only one or the other.

**Comment by swift-kim** [comment]
Hmm. Have you tried uninstalling the plugin? If the scores are still not displayed correctly even after uninstalling the plugin, it's likely the problem is with the game itself but not this plugin. In this case you might need to delete the `PlayerData.dat` (caution: all your stats will be deleted so you'd just rename the file to something like `PlayerData.dat.old`) or reinstall the game. Otherwise, if the scores are displayed correctly after uninstalling the plugin, something has gone wrong with the plugin and I will need the log file I mentioned in the above to diagnose the problem.
